TestRunner.psm1

```diff
@@ -798,7 +795,7 @@ function Get-RunnerOutputDir {
         $Config
     )
     $outputDir = Get-OutputDir -Config $Config
-    $runnerDir = Join-Path -Path $outputDir -ChildPath "runner"
+    $runnerDir = Join-Path -Path $outputDir -ChildPath (Get-Date -Format "yyyy-MM-dd-HH-mm-ss")
```

Comment:
I wanted the test results to go into a dated directory, like with the paging tests.